pad_api_data/pad_etl/common/dungeon_parse.py

```diff
@@ -1,12 +1,21 @@
-# A small object to hold floor modifier information
+from dungeon_maps import btypeChart, battrChart
+
+
 class Modifier:
     def __init__(self):
         self.requiredDungeon = 0
-        self.modifiers = []
+        self.remainingModifiers = []
         self.entryRequirement = "None"
+        self.modifiers = {}
+        self.messages = []
+        self.fixedTeam = {}
+        self.enhancedType = "None"
+        self.enhancedAttribute = "None"
```

Comment:
Single quotes are preferred; regardless the style should be consistent in the same file.